javascript/modules/client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-client",
   "author": "Aragon Association",
-  "version": "0.8.0-alpha",
+  "version": "0.8.1-alpha",
```

Comment:
These changes feel like a breaking change, not a fix. Since we are in alpha stage, I would suggest bumping to 0.9.0-alpha